scripts/kms/constitution_set.sh

```diff
@@ -10,7 +10,8 @@ constitution-set() {
   CONSTITUTION_PATH=$1
```

Comment:
can we test if $1 was passed? Will also be clearer to understand the scripts

Reply:
This file is completely changed in #242 so I don't think it's worth changing here especially since it's unrelated to the change in this PR